glom/tutorial.py

```diff
@@ -16,14 +16,24 @@
 
 What does nested data looks like? In its most basic form::
 
-  >>> data = {'a': {'b': {'c': 'd'}}}
+  >>> data = {
+  ...     'a': {
+  ...         'b': {
+  ...             'c': 'd'
+  ...         }
+  ...     }
+  ... }
```

Comment:
hmm for these really short ones just starting out, I think it might be easier to see the translation to glom paths when it's all on one line. Once we get into planets and we start blowing out the width, maybe we switch then? What do you think?

Reply:
yeah i think this straight-forward example is fine to be on one line 👍